assets/tests/util/streetViewUrl.test.ts

```diff
@@ -0,0 +1,26 @@
+import { streetViewUrl } from "../../src/util/streetViewUrl"
+
+describe("streetViewUrl", () => {
+  test("returns a url with the given lat, lon, and bearing", () => {
+    expect(
+      streetViewUrl({
+        latitude: 42.36014757,
+        longitude: -71.09505978,
+        bearing: 131,
+      })
+    ).toEqual(
+      "https://www.google.com/maps/@?api=1&map_action=pano&viewpoint=42.36014757%2C-71.09505978&heading=131&pitch=0&fov=80"
+    )
+  })
+
+  test("returns with default heading of 0 when no bearing given", () => {
```

Comment:
**nitpick (non-blocking):** test name should be updated to reflect change to functionality.